src/views/BotDetail.vue

```diff
@@ -64,7 +64,9 @@
                     :rules="[val => val && val.length > 0 || '必須項目です']")
                   q-input(label="説明" stack-label v-model="description" :readonly="!editing" autogrow type="textarea" hide-hint hint="使用用途等を入力してください"
                     :rules="[val => val && val.length > 0 || '必須項目です']")
-                  q-input(label="BOTサーバーエンドポイント" stack-label v-model="endpoint" :readonly="!editing" hide-hint  hint="traQのイベント送信先のURLを入力してください"
+                  q-select(label="動作モード" stack-label v-model="mode" :readonly="!editing" hide-hint hint="動作モードを選択してください"
+                    :options="modeOptions")
+                  q-input(label="BOTサーバーエンドポイント" stack-label v-model="endpoint" :readonly="!editing" v-if="mode === 'HTTP'" hide-hint  hint="traQのイベント送信先のURLを入力してください"
```

Comment:
```suggestion
                  q-input(v-if="mode === 'HTTP'" label="BOTサーバーエンドポイント" stack-label v-model="endpoint" :readonly="!editing" hide-hint  hint="traQのイベント送信先のURLを入力してください"
```
`v-if`は最初にしてほしい :pray: